tests/python/testing.py

```diff
@@ -33,8 +32,13 @@ def no_sklearn():
 
 
 def no_dask():
-    return {'condition': not DASK_INSTALLED,
-            'reason': 'Dask is not installed'}
+    reason = "dask/distributed is not installed."
+    try:
+        import dask             # noqa
+        import distributed      # noqa
```

Comment:
I'm not sure if importing dask to check its existence is necessarily a good idea. Please see #6521 and #6522

Reply:
This is in testing module, are we worrying about import?